### PR TITLE
Add global --http-debug flag

### PR DESCRIPTION
**Pull request by kelseiv**
https://github.com/influxdata/docs-v2/issues/2947

For any curious (re changes in the v2.0 directory), I started down the rabbit hole of removing duplicate-oss tags in cloud docs, and decided it wasn't worth the effort for such a short-term issue.

### CODE REVIEW

File: content/influxdb/v2.0/reference/cli/influx/auth/list.md [hunk 1]
```diff
@@ -21,21 +21,22 @@ influx auth list [flags]
...
+|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+| `-i` | `--id`            | Authentication token ID                                               |   string   |                       |
+|      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |
+| `-o` | `--org`           | Organization name (mutually exclusive with `--org-id`)                |   string   |                       |
+|      | `--org-id`        | Organization ID (mutually exclusive with `--org`)                     |   string   |                       |
+|      | `--skip-verify`   | Skip TLS certificate verification                                     |            |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/auth/list.md:36]: i think skip-verify also has an environment variable now as of https://github.com/influxdata/influx-cli/pull/208 unless that's addressed in another PR.

File: content/influxdb/v2.0/reference/cli/influx/task/delete.md [hunk 2]
```diff
@@ -16,17 +16,18 @@ influx task delete [flags]
...
+| :--- | :---------------- | :-------------------------------------------------------------------- | :--------: | :-------------------- |
+| `-c` | `--active-config` | CLI configuration to use for command                                  |   string   |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) |   string   | `INFLUX_CONFIGS_PATH` |
+| `-h` | `--help`          | Help for the `delete` command                                         |            |                       |
+|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+| `-i` | `--id`            | ({{< req >}}) Task ID                                                 |   string   |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/task/delete.md:27]: what's with the `({{< req >}})` in this line? i don't see it elsewhere.

File: content/influxdb/v2.0/reference/cli/influx/task/run/list.md [hunk 3]
```diff
@@ -21,21 +21,22 @@ influx task run list [flags]
...
+|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+|      | `--json`          | Output data as JSON (default `false`)                                 |            | `INFLUX_OUTPUT_JSON`  |
+|      | `--limit`         | Limit the number of results                                           |  integer   |                       |
+|      | `--run-id`        | Run ID                                                                |   string   |                       |
+|      | `--skip-verify`   | Skip TLS certificate verification                                     |            |                       |
+|      | `--task-id`       | ({{< req >}}) Task ID                                                 |   string   |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/task/run/list.md:38]: this one too.

File: content/influxdb/v2.0/reference/cli/influx/task/update.md [hunk 4]
```diff
@@ -16,19 +16,20 @@ influx task update [task literal] [flags]
...
+| `-c` | `--active-config` | CLI configuration to use for command                                  |   string   |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) |   string   | `INFLUX_CONFIGS_PATH` |
+| `-f` | `--file`          | Path to Flux script file                                              |   string   |                       |
+| `-h` | `--help`          | Help for the `update` command                                         |            |                       |
+|      | `--hide-headers`  | Hide table headers (default `false`)                                  |            | `INFLUX_HIDE_HEADERS` |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+| `-i` | `--id`            | ({{< req >}}) Task ID                                                 |   string   |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/task/update.md:28]: here

File: content/influxdb/v2.0/reference/cli/influx/task/run/retry.md [hunk 5]
```diff
@@ -16,16 +16,17 @@ influx task run retry [flags]
...
+| Flag |                   | Description                                                           | Input type | {{< cli/mapped >}}    |
+| :--- | :---------------- | :-------------------------------------------------------------------- | :--------: | :-------------------- |
+| `-c` | `--active-config` | CLI configuration to use for command                                  |   string   |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`) |   string   | `INFLUX_CONFIGS_PATH` |
+| `-h` | `--help`          | Help for the `retry` command                                          |            |                       |
+|      | `--host`          | HTTP address of InfluxDB (default `http://localhost:8086`)            |   string   | `INFLUX_HOST`         |
+|      | `--http-debug`    | Inspect communication with InfluxDB servers.                          |   string   |                       |
+| `-r` | `--run-id`        | ({{< req >}}) Run ID                                                  |   string   |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/task/run/retry.md:26]: here

File: content/influxdb/v2.0/reference/cli/influx/task/run/retry.md [hunk 6]
```diff
@@ -16,16 +16,17 @@ influx task run retry [flags]
...
+|      | `--skip-verify`   | Skip TLS certificate verification                                     |            |                       |
+| `-i` | `--task-id`       | ({{< req >}}) Task ID                                                 |   string   |                       |
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/task/run/retry.md:28]: here

File: content/influxdb/v2.0/reference/cli/influx/auth/list.md (hunk 1 above)
Comment (danxmoran) [reply content/influxdb/v2.0/reference/cli/influx/auth/list.md:36]: It's in #2944

File: content/influxdb/v2.0/reference/cli/influx/config/create.md [hunk 7]
```diff
@@ -24,6 +24,7 @@ influx config create [flags]
 | `-h` | `--help`         | Help for the `create` command                                |             |                       |
 |      | `--hide-headers` | Hide table headers (default `false`)                         |             | `INFLUX_HIDE_HEADERS` |
 | `-u` | `--host-url`     | ({{< req >}}) Connection URL for the new configuration.     | string      |                       |
+|      | `--http-debug`   | Inspect communication with InfluxDB servers.                 | string      |                      |
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influx/config/create.md:27]: The `config` commands don't support this flag since they're CLI-only

**Review by sanderson — APPROVED** [review]

File: content/influxdb/v2.0/reference/cli/influx/task/delete.md (hunk 2 above)
Comment (kelseiv) [reply content/influxdb/v2.0/reference/cli/influx/task/delete.md:27]: shortcode to identify required parameters
https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#required-elements

**Comment by kelseiv** [comment]
oops, @sanderson didn't mean to re-request for you :-)

**Review by danxmoran — APPROVED** [review]

**Review by sanderson — APPROVED** [review]

**Pull request merged**